testnets/_IBC/prysm-axelar.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "prysm-devnet-1",
```

Comment:
for all ibc files, the referenced chain names (both in the `chain_name` property and in the file name) must reference the actualy chain_name that the chain Registry uses. E.g., in this case, it would be 'axelartestnet' and 'prysmdevnet'.
Also, the chain_1 and chain_2 must be in alphabetical order, so prysmdevnet should be chain_2 for most of these.